ppa/archive/templates/archive/add_to_collection.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'admin/base_site.html'  %}
+{% load i18n grp_tags %}
+{# Adapts Grappelli form styles for use in an intermediate view #}
+{% block title %} Bulk Add to Collections | {% get_site_title %} {% endblock %}
+{% block breadcrumbs %}
+    {% if not is_popup %}
+        <ul class="grp-horizontal-list">
+            <li><a href="{% url 'admin:index' %}">{% trans "Home" %}</a></li>
+            <li><a href="{% url 'admin:app_list' app_label='archive' %}">Archive</a></li>
+            <li><a href="{% url 'admin:archive_digitizedwork_changelist'%}">Digitized Works</a></li>
+            <li>Add DigitizedWorks to Collections</li>
+        </ul>
+    {% endif %}
+{% endblock %}
+
+{% block content_title %}
+    <h1>Add DigitizedWorks to Collections</h1>
```

Comment:
minor quibble: `DigitizedWorks` is the name of the model; it would be better to use the verbose/human readable version of that here and elsewhere on user-facing pages.